tasks/AddClues.md

```diff
@@ -0,0 +1,27 @@
+# Labyrinth: Add clues to lead to the various parts of the labyrinth !!
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+To make the game more interesting, please add a functionality to add music when the user enters a tile,
+
+Requirements:
+- In the `tiles.js` file edit the `alterInfo` tag of various tiles to lead the user to the various parts of the labyrinth (e.g. new year tile, etc)
+- Note: the clues you give are logical and tangible
```

Comment:
I do not know what logical and tangible means. If you are a mentor and you have a disagreement with a student about whether this is tangible and logical, how do you both get to an agreement?

Reply:
hmm wait ill make some changes...

Reply:
Maybe you can give examples 3+ and from there we abstact.

Reply:
examples will also guide the people who claim.